packages/Dendrogram/src/workers/distance-functions.ts

```diff
@@ -0,0 +1,35 @@
+import {distance} from 'fastest-levenshtein';
+
+function levensteinDistance(a: string, b: string): number {
+  return distance(a, b);
+};
+
+function numericDistance(a: number, b: number): number {
+  return Math.abs(a - b);
+}
+
+function hammingDistance(a: string, b: string) {
```

Comment:
All function return types should be annotated; void is the only exception from the rule